src/bot.ts

```diff
@@ -53,9 +55,9 @@ export const startBot = async (): Promise<void> => {
       initMessageCreate(client, logger, message);
     });
     client.login();
-  } catch (e) {
-    console.log('Bot failure');
-    console.log(e);
+  } catch (e: unknown) {
+    logger.error('Uh oh, something went wrong when initializing the bot!');
+    logger.error(e);
```

Comment:
Could these two lines be combined into one `logger.error`, but separated with `\n`?